webapp-django/crashstats/topcrashers/views.py

```diff
@@ -66,11 +66,19 @@ def get_date_boundaries(parameters):
     return (greater_than, lower_than)
 
 
+def datetime_to_build_id(date):
+    """Return a build_id-like string from a datetime. """
+    return date.strftime('%Y%m%d%H%M%S')
+
+
 def get_topcrashers_results(**kwargs):
-    '''Return the results of a search. '''
+    """Return the results of a search. """
     results = []
 
     params = kwargs
+    build_mode = params.pop('_range_type') == 'build'
```

Comment:
Also, `build_mode` ends up with a boolean value. I think the related code reads better if it was called `is_build`.